code/game/gamemodes/miniantags/demons/slaughter demon/slaughter_demon.dm

```diff
@@ -0,0 +1,310 @@
+/mob/living/simple_animal/demon/slaughter_demon
+	name = "slaughter demon"
+	real_name = "slaughter demon"
+	desc = "A large, menacing creature covered in armored black scales. You should run."
+	maxHealth = 400
+	health = 400
+	speak = list("ire", "ego", "nahlizet", "certum", "veri", "jatkaa", "balaq", "mgar", "karazet", "geeri", "orkan", "allaq")
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "daemon"
+	icon_living = "daemon"
+	obj_damage = 50
+	melee_damage_lower = 30
+	melee_damage_upper = 30
+	environment_smash = ENVIRONMENT_SMASH_RWALLS
+	armour_penetration_percentage = 100
+	var/channeling = FALSE
+	var/lasthealtime = 0
+	var/phased = TRUE
+	var/phaseoutchaneltime = 0
+	var/phaseinchaneltime = 0
+	var/stat_amplification = 0.5
+	var/max_amplification_stacks = 0.5 //50% max
+	var/last_stat_decrease = 0
+	var/decay_time = 0
+	var/boost = 0
+	var/feast_sound = 'sound/misc/demon_consume.ogg'
+	var/devoured = 0
+	var/mindsabsorbed = 0
+	var/list/consumed_mobs = list()
+	var/list/nearby_mortals = list()
+	var/list/bloodspots = list()
+	var/cooldown = 0
+	var/gorecooldown = 0
+	var/vialspawned = FALSE
+	var/turf/channel_target = 0
+	var/obj/effect/proc_holder/spell/demon_slam/slam_holder
+	var/obj/effect/proc_holder/spell/demon_charge/charge_holder
```

Comment:
Take another look at these and document what you think could use documentation.
Oh and some of these are unused i think, look for those too.

Reply:
Please also snake_case the rest of the new variables here that use more than one word for the name.
`mind_absorbed, blood_spots, etc.`